src/utilities/filetypes/EpwFile.cpp

```diff
@@ -3967,6 +3974,31 @@ bool EpwFile::parse(std::istream& ifs, bool storeData) {
     }
   }
 
+  for (unsigned int i = 0; i < epw_strings.size(); i++) {
```

Comment:
Do this after the previous loop, and not inside of it, because we first need to get `realYear`.